tests/bears/BearTest.py

```diff
@@ -180,6 +180,10 @@ def test_detect_and_fix_aspects(self):
         assert (CommitMessage.Shortlog.TrailingPeriod in
                 aspectsTestBear.aspects['fix'])
 
+    def test_aspectlist_bear(self):
+        assert aspectsTestBear.aspects['fix'].bear == aspectsTestBear
```

Comment:
Use `is` to assert the same object reference https://stackoverflow.com/questions/132988/is-there-a-difference-between-and-is-in-python